spring-rabbit/src/main/java/org/springframework/amqp/rabbit/listener/DirectMessageListenerContainer.java

```diff
@@ -248,19 +230,12 @@ public boolean removeQueueNames(String... queueNames) {
 		return super.removeQueueNames(queueNames);
 	}
 
-	@Override
-	public boolean removeQueues(Queue... queues) {
-		removeQueues(Arrays.stream(queues)
```

Comment:
The same here. To the `private void removeQueues(Stream<String> queueNames) `

What am I missing, please ?